config.json

```diff
@@ -128,6 +128,14 @@
           "floating_point_numbers"
         ]
       },
+      {
+        "slug": "bob",
+        "name": "Bob",
+        "uuid": "6d8927ca-2aaa-4af4-bb38-c307b2105e41",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```

Comment:
Is this intentionally marked easier than, say, resistor duo?

Reply:
Not intentional. Now increased to 3, to match resistor-color-duo.

I think resistor-color difficulty rating was also too low on this track (https://github.com/exercism/x86-64-assembly/pull/241).